doc/lua_api.txt

```diff
@@ -2495,16 +2495,26 @@ Elements
         * `span=<value>`: number of following columns to affect
           (default: infinite).
 
-### `style[<name 1>,<name 2>,...;<prop1>;<prop2>;...]`
+### `style[<name 1>,<name 2>;<prop1>;<prop2>;...]`
+### `style[<name 1>:<state>,<name 2>:state;<prop1>;<prop2>;...]`
 
 * Set the style for the named element(s) `name`.
+* `state` is a list of states separated by the `+` character.
+    * If a state that the element supports is provided, the style will only
```

Comment:
```suggestion
    * If a state is provided, the style will only take affect if the element
      is in that state. If an element doesn't support a state, it will never be
      in that state.
```

Reply:
`If an element doesn't support a state, it will never be in that state.`
Ah yes, the floor is made out of floor.

The entire sentence could be removed for being obvious.

Reply:
yeah, I don't know why the original text specified that